packages/circuits/circuits/proveReputation.circom

```diff
@@ -177,7 +196,8 @@ template ProveReputation(STATE_TREE_DEPTH, EPOCH_KEY_NONCE_PER_EPOCH, MAX_REPUTA
 
     /* 4. Check nonce and output epoch key */
 
-    nonce \ EPOCH_KEY_NONCE_PER_EPOCH === 0;
+    signal epknonce_divided <-- nonce \ EPOCH_KEY_NONCE_PER_EPOCH;
+    epknonce_divided === 0;
```

Comment:
This check doesn't work, need to use a `LessThan` component. You can use `LessThan(8)` because we constrain the value of `nonce` to 8 bits above.

Reply:
fix here [7b47f60](https://github.com/Unirep/Unirep/pull/295/commits/7b47f602654856d80bec74ea01c9288b0a973422)
and also I proposed that `proveGraffiti` can be in `control1` so that the `proveReputation` can use `epochKeyLite` template

Reply:
I think that's fine, just make sure to update the [`decodeReputationSignals`](https://github.com/Unirep/Unirep/blob/main/packages/contracts/contracts/Unirep.sol#L464) function.